pg_replicate/src/pipeline/sinks/bigquery.rs

```diff
@@ -79,6 +79,7 @@ impl BigQueryBatchSink {
         })
     }
 
+    #[expect(clippy::result_large_err)]
```

Comment:
Suppressing the large error variant clippy warning for now. Will handle in a separate PR.